src/Smd/Service.php

```diff
@@ -221,10 +235,8 @@ public function setTarget($target)
 
     /**
      * Get service target.
-     *
-     * @return string
      */
-    public function getTarget()
+    public function getTarget(): ?string
```

Comment:
Careful: adding return types to open classes is a BC break - please avoid doing this on non-`final` classes

Reply:
Agreed, reverted.